doc/FAQ.md

```diff
@@ -171,9 +171,7 @@ Example: Friendica Support
 <a name="clients"></a>
 ### What friendica clients can I use?
 
-Friendica is using a [Twitter/GNU Social compatible API](help/api), which means you can use any Twitter/GNU Social client for your platform as long as you can change the API path in its settings.
-Since the 2021.06 release, Friendica also supports the [Mastodon API](help/API-Mastodon) which works with many Mastodon clients.
-Here is a list of known working clients:
+Friendica supports [Mastodon API](help/API-Mastodon) and [Twitter API](help/api). This means you can use some of the Mastodon and Twitter clients for Friendica. The available features are client specific and may differ.
```

Comment:
Did we remove the GNU Social extension from the Twitter like API that is supported by Friendica?

Reply:
https://github.com/friendica/friendica/pull/11010